next.config.js

```diff
@@ -19,6 +19,10 @@ const moduleExports =  {
     ]
   },
 
+  env: {
+    releaseVersion: 'full', // feature flag - mvp or full
```

Comment:
Could this be validated during startup to ensure it is a valid value?
This could avoid unexpected behaviour(s) whilst running.

Reply:
We had a discussion on this and we decided that any other value apart from mvp would be the full version.